mkdocs.yml

```
@@ -19,14 +19,24 @@ extra:
   social:
     - icon: fontawesome/brands/github
       link: https://github.com/engula
+    - icon: fontawesome/brands/twitter
+      link: https://twitter.com/engulaio
   analytics:
     provider: google
     property: G-FHW7MJB9NB
 
 plugins:
   - git-revision-date
 
+markdown_extensions:
+  - pymdownx.highlight
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences
+
 nav:
   - Home: index.md
   - Posts:
-    - "Demo 1": posts/demo-1.md
\ No newline at end of file
+    - "Demo 1": posts/demo-1.md
+    - "Release 0.2": posts/release-0.2.md
+    - "Tutorial 0.2": posts/tutorial-0.2.md
```

```suggestion
    - "Tutorial 0.2": posts/tutorial-0.2.md
    - "Release 0.2": posts/release-0.2.md
    - "Demo 1": posts/demo-1.md
```

a descending order can help readers focus on latest posts.